include/drivers/pwm.h

```diff
@@ -63,6 +63,7 @@ __subsystem struct pwm_driver_api {
  * @param dev Pointer to the device structure for the driver instance.
  * @param pwm PWM pin.
  * @param period Period (in clock cycle) set to the PWM. HW specific.
+ * 		 Pass 0 to disables the signal.
```

Comment:
We should also describe what "disabled" is in this context (does it tri-state the pin, tie it high, tie it low, ...)?
I believe it should tie it to "inactive" according to the specified polarity flag (e.g. low for `PWM_POLARITY_NORMAL`, high for `PWM_POLARITY_INVERTED`).

Reply:
I agree. Tie it to "inactive" should be a save state. Tri-state may lead to unwanted switching activity.

Reply:
well, I would assume if someone wants to "tri-state" the pin, one should look into reconfiguring pinmux/pinctrl -- after all, that should be governed by the pin/port controller.

As for the "inactive" state (i.e. non-tri-state), I don't understand why we need to play with period here.

If POL = Active High, PULSE = 0 means LOW, PULSE = PERIOD means HIGH.
If POL = Active Low, PULSe = 0 means HIGH, PULSE = PERIOD means LOW.